mysql/datadog_checks/mysql/mysql.py

```diff
@@ -851,7 +866,39 @@ def _query_size_per_schema(self, db):
 
                 return schema_size
         except (pymysql.err.InternalError, pymysql.err.OperationalError) as e:
-            self.warning("Avg exec time performance metrics unavailable at this time: %s", e)
+            self.warning("Size of schemas metrics unavailable at this time: %s", e)
+
+        return {}
+
+    def _query_size_per_table(self, db, extra=False):
+        try:
+            with closing(db.cursor()) as cursor:
+                if extra:
+                    cursor.execute(SQL_QUERY_EXTRA_TABLE_SIZE)
+                else:
+                    cursor.execute(SQL_QUERY_TABLE_SIZE)
+
+                if cursor.rowcount < 1:
+                    self.warning("Failed to fetch records from the information schema 'tables' table.")
+                    return None
+
+                table_index_size = {}
+                table_data_size = {}
+                for row in cursor.fetchall():
+                    table_schema = str(row[0])
+                    table_name = str(row[1])
+                    index_size = long(row[2])
+                    data_size = long(row[3])
```

Comment:
I don't think you should cast these two values to `long`s here, I think its fine to not cast it and let python leave it as a decimal. That way we don't lose information (especially for smaller tables), since the query gives back an answer with precision to several decimal places